### PR TITLE
[WIP] CHANGELOG.md updated for v0.8.0

### PR DESCRIPTION
**Pull request by ttmc**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,10 +16,69 @@ For reference, the possible headings are:
...
+- Wrote a formal schema for the JSON structure of transactions. Transactions are now checked against that schema. [Pull Request #798](https://github.com/bigchaindb/bigchaindb/pull/798)
+
+### Changed
+- CREATE transactions must now be signed by all `owners_before` (rather than by a federation node). [Pull Request #794](https://github.com/bigchaindb/bigchaindb/pull/794)
+- The user-provided timestamp was removed from the transaction data model (schema). [Pull Request #817](https://github.com/bigchaindb/bigchaindb/pull/817)
+- `get_transaction()` will now return a transaction from the backlog, even if there are copies of the transaction in invalid blocks. [Pull Request #793](https://github.com/bigchaindb/bigchaindb/pull/793)
+- Several pull requests to translate RethinkDB database calls into abstract/generic database calls, and to implement those abstract calls for two backends: RethinkDB and MongoDB. Pull Requests
```
Comment (libscott) [inline CHANGELOG.md]: Wording seems off (code is translating?), sounds like the code is performing translation. How about:

> Several pull requests to introduce a generalized database interface, to move RethinkDB calls into a separate implementation of that interface, and work on a new implementation for a MongoDB backend.

Could also leave as is.

**Comment by libscott** [comment]
Looks good as far as spelling and wording goes, save one quite optional comment that I added.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,10 +16,69 @@ For reference, the possible headings are:
...
+- `get_transaction_by_metadata_id` now ignores invalid transactions. [Pull Request #811](https://github.com/bigchaindb/bigchaindb/pull/811)
+- Updates to the configs and scripts for deploying a test network on AWS. The example config file deploys virtual machines running Ubuntu 16.04 now. Pull Requests 
+[#771](https://github.com/bigchaindb/bigchaindb/pull/771),
+[#813](https://github.com/bigchaindb/bigchaindb/pull/813)
+- Changed logging of transactions on block creation so now it just says the length of the list of transactions, rather than listing all the transactions. [Pull Request #861](https://github.com/bigchaindb/bigchaindb/pull/861)
+
+### Fixed
+- Equality checks with AssetLinks. [Pull Request #825](https://github.com/bigchaindb/bigchaindb/pull/825)
+- Bug in `bigchaindb load`. [Pull Request #824](https://github.com/bigchaindb/bigchaindb/pull/824)
+- Two issus found with timestamp indexes. [Pull Request #816](https://github.com/bigchaindb/bigchaindb/pull/816)
```
Comment (sbellem) [inline CHANGELOG.md]: issus --> issues

Comment (libscott) [inline CHANGELOG.md]: Welp :) looks like I can't spell! :D

File: CHANGELOG.md (hunk 1 above)
Comment (ttmc) [inline CHANGELOG.md]: I liked your version, so I used it and made a minor edit.

File: CHANGELOG.md (hunk 2 above)
Comment (ttmc) [inline CHANGELOG.md]: Thanks!

**Pull request merged**